README.md

```diff
@@ -9,6 +9,7 @@
 [![Contributor Covenant](https://img.shields.io/badge/Contributor%20Covenant-2.1-4baaaa.svg)][CoC]
 [![pre-commit](https://img.shields.io/badge/pre--commit-enabled-brightgreen?logo=pre-commit)][pre-commit]
 [![Downloads](https://pepy.tech/badge/phylum/month)][downloads]
+[![Discord](https://img.shields.io/discord/1070071012353376387?logo=discord)][discord_invite]
```

Comment:
Would have probably been faster to add this in a separate PR, but it's minor enough to where I don't really mind.

Reply:
Roger that. There is not a real rush to get this in. It was added as just another "benign" change to this "demonstration" PR (but will eventually get merged in).